plugins/glustershd/init.go

```diff
@@ -32,10 +33,17 @@ func (p *Plugin) RestRoutes() route.Routes {
 			Version:      1,
 			ResponseType: utils.GetTypeString(([]glustershdapi.BrickHealInfo)(nil)),
 			HandlerFunc:  selfhealInfoHandler},
+		route.Route{
+			Name:        "SelfHeal",
+			Method:      "POST",
+			Pattern:     "/volumes/{name}/heal",
```

Comment:
Use `{volname}` for consistency as it's used everywhere else. See `glusterd2/commands/volumes/commands.go`

Please use it other self heal handlers too.